Resources/Prototypes/Entities/Objects/Misc/paper.yml

```diff
@@ -351,11 +351,20 @@
     damage:
       types:
         Piercing: 15
+    soundHit:
+      path: /Audio/Weapons/bladeslice.ogg
   - type: Tool
     qualities:
       - Screwing
     useSound:
       collection: Screwdriver
+  - type: UnpoweredFlashlight
+  - type: PointLight
+    enabled: false
+    mask: /Textures/Effects/LightMasks/cone.png
+    radius: 12
+    energy: 2
+    autoRot: true
```

Comment:
Don't give it a flashlight, especially not a forever powered one. Like Hanzdegloker said it already does so much

Reply:
MAYBE i will think aboiut it if its VERY weak

Reply:
OK well I personally don't agree but that's fair 😆!

I wanted to make it somewhat powerful because it has to be better than the PDAs light otherwise there isn't a point. My only other idea was to make a new light mask that has a significantly smaller angle so it acts more like a penlight. Should I bother trying to do that or will it get denied?

Reply:
I would vote to just remove it

Reply:
OK sounds good then. I'll update the PR just to have the noise now :)